localstack/services/sqs/provider.py

```diff
@@ -254,27 +264,16 @@ def update_last_modified(self, timestamp: int = None):
 
         self.attributes[QueueAttributeName.LastModifiedTimestamp] = str(timestamp)
 
-    @property
-    def name(self):
-        return self.key.name
-
-    @property
-    def owner(self):
-        return self.key.account_id
-
-    @property
-    def arn(self) -> str:
-        return f"arn:aws:sqs:{self.key.region}:{self.key.account_id}:{self.key.name}"
-
     def url(self, context: RequestContext) -> str:
         """Return queue URL using either SQS_PORT_EXTERNAL (if configured), or based on the 'Host' request header"""
         host_url = context.request.host_url
         if config.SQS_PORT_EXTERNAL:
             host_url = external_service_url("sqs")
+
         return "{host}/{account_id}/{name}".format(
             host=host_url.rstrip("/"),
-            account_id=self.key.account_id,
-            name=self.key.name,
+            account_id=context.account_id,
+            name=self.name,
```

Comment:
this is where we need the account and region information from the queue in #5928